src/wallet/index.ts

```diff
@@ -53,13 +54,17 @@ class Wallet {
    *
    * @param publicKey - The public key for the account.
    * @param privateKey - The private key used for signing transactions for the account.
-   * @param seed - (Optional) The seed used to derive the account keys.
+   * @param opts - (Optional) The seed and classicAddress (corresponding to its Regular Key Pair) for the account.
```

Comment:
`opts` seems unrelated to regular keys. Maybe `regularKeyOpts` would highlight that relationship?

Reply:
We use `options` or `opts` for additional optional options, so that you don't need to worry about the order of all the optional stuff and you just include what you need to use.